Description: ignore missing plugins
Author: Pirate Praveen <praveen@debian.org>
Forwarded: not-needed
Reviewed-By: Xavier Guimard <yadd@debian.org>
Last-Update: 2020-02-22

--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -74,10 +74,10 @@
 	// tasks
 	grunt.loadNpmTasks("grunt-contrib-concat");
 	grunt.loadNpmTasks("grunt-contrib-nodeunit");
-	grunt.loadNpmTasks("grunt-contrib-watch");
+	//grunt.loadNpmTasks("grunt-contrib-watch");
 	grunt.loadNpmTasks("grunt-contrib-uglify");
 	grunt.loadNpmTasks("grunt-babel");
-	grunt.loadNpmTasks("grunt-eslint");
+	//grunt.loadNpmTasks("grunt-eslint");
 	grunt.task.registerTask("babili", "Minifies ES2016+ code", function () {
 		const data = fs.readFileSync(path.join(__dirname, "lib", "filesize.es6.js"), "utf8"),
 			minified = require("babel-core").transform(data, {
